content/en/tutorials/first-person-movement.md

```diff
@@ -9,36 +9,47 @@ thumb: https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/4058
 
 This is an application that implements first person character movement.
 
-The scene setup for this controller is important as your character must have a rigidbody and collision component in addition to the script attached. In addition, the script supports adding a camera entity as a child of the Player, if no camera entity is present a new entity is created. See the full scene setup in the [Tutorial Project][1].
+The scene setup for this controller is important as your character must have a rigidbody and collision component in addition to the script attached. In addition, the script supports adding a camera entity as a child of the Player and manually set in the inspector. If no camera entity is present a new entity is created. 
+
+See the full scene setup in the [Tutorial Project][1].
 
 The script below performs the following functions:
 
 * Listen for mouse and keyboard input
 * Update a camera entity from the mouse input
 * Apply forces to move the player entity around the scene
 
-Note, the player's velocity is never set directly but he is moved by apply forces. To limit the maximum velocity, we have linear damping applied on the rigidbody component.
+Note, the player's velocity is never set directly but he is moved by apply forces. 
```

Comment:
apply -> applied?

Reply:
Ah, I think the old text was referring to the function. I will fix that up to make it more clear